crates/transcode/src/lib.rs

```diff
@@ -277,6 +277,13 @@ impl ContractMessageTranscoder {
             args.push((Value::String(name), value));
         }
 
+        if !data.is_empty() {
```

Comment:
There is one other usage of this`decode_contract_event` method, for decoding emitted contract events: https://github.com/paritytech/cargo-contract/blob/0de5b430c22361b33abb5873fec8823b95e69108/crates/cargo-contract/src/cmd/extrinsics/events.rs#L215

I believe it should continue to work, but it is worth double checking

Reply:
Checked. It used when parsing events from the result of `call` or `instantiate` commands. Everything works.